charts/rhtap-backing-services/templates/openshift-acs/job-stackrox-api.yaml

```diff
@@ -0,0 +1,58 @@
+{{- if .Values.backingServices.acs.enabled -}}
+  {{- $acs := .Values.backingServices.acs -}}
+  {{- $name := printf "%s-post-deploy-%d" $acs.name .Release.Revision }}
+#
+# Generates a token for StackRox API.
+#
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  annotations:
+    helm.sh/hook: post-install,post-upgrade
+    helm.sh/hook-delete-policy: hook-succeeded
+  labels:
+    {{- include "rhtap-backing-services.labels" . | nindent 4 }}
+  namespace: {{ $acs.namespace }}
```

Comment:
That job has to be run in the rhtap namespace as the `rhtap-acs-integration` is expected to be there.

Reply:
Cool, thanks.